src/ralph/admin/sites.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict
+import six
+
+from django.apps import apps
+from django.conf.urls import url
+from django.contrib.admin.sites import AdminSite
+from django.core.urlresolvers import NoReverseMatch, reverse
+from django.template.response import TemplateResponse
+from django.utils.text import capfirst
+
+
+class RalphAdminMixin(object):
```

Comment:
there are 2 `RalphAdminMixin` - one here and one in `admin/mixins.py` - it may be misleading so could you rename it somehow?